src/components/app/App.tsx

```diff
@@ -0,0 +1,8 @@
+import { TaskListContainer } from "../taskListContainer/TaskListContainer";
+import "./App.css";
+
+export const App = (): JSX.Element => (
```

Comment:
No specific reason other than vscode recommended it as the inferred type

Reply:
HA! Wrong answer, young lady! But also right 😊

A ReactElement is an object with a type and props.

JSX.Element is a ReactElement, with the generic type for props and type being any. JSX is a global namespace that then gets implemented by various libraries.

Since we are using the library React, let's use the specific type the way it sets the generic namespace.
Next time you add a commit, please replace those `JSX.Element` with `ReactElement`

Reply:
Ohh I see, so it's best to use ReactElement because the global namespace for JSX might get overwritten, yeah? I'll update.

Reply:
Yas. Let's use the specific implementation of the generic namespace done by React.
Not a big deal, but since we are learning ...